DEVELOP.md

```diff
@@ -1,11 +1,30 @@
 # Contributing to the RAPIDS devcontainers
 
-From the official devcontainer [documentation on Features](https://containers.dev/implementors/features/):
-> Development container "Features" are self-contained, shareable units of installation code and development container configuration.
+This document contains implementation details and design overviews for
+[rapidsai/devcontainers](https://github.com/rapidsai/devcontainers) as a
+centralized source of common scripts and patterns for providing devcontainers.
+
+For the user-level overview providing instructions on how to use the
+devcontainer as a development environment, see
+[USAGE_IN_PROJECT.md](USAGE_IN_PROJECT.md)
+
+For the project maintainer-level overview providing instructions on how to add
+and change .devcontainer.json to suit your project, see [USAGE.md](USAGE.md)
+
+The code in this repository fits into a few main categories:
+1. Features
+1. GitHub Actions automations
+1. Scripts
+1. matrix.yml
+1. Dockerfiles
```

Comment:
I changed to this because I find correct numbers to be cumbersome when editing a document. If you insert a new step or rearrange steps, you have to manually correct numbering. To me, that is more tedious than the lack of meaningful numbers when viewing the raw markdown.

I'm fine with conforming if there's a strong preference at NVIDIA one way or the other. We could also go by [Google's style guide](https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists), which prefers correct numbers for short, simple lists, but lazy numbering otherwise. I think I'd classify this list as short, because it is short in length and its content doesn't seem likely to change much over time.

Reply:
FYI: In past conversations where this has come up, we have always settled on actual numbers 1, 2, 3, …. That’s my preference here as well.

Reply:
If appropriate, you can use unordered lists to avoid renumbering.